src/senders/Sendsmaily.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace nuffic\subscribe\senders;
+
+use nuffic\sendsmaily\ApiClient;
+use Yii;
+use yii\base\Model;
+
+class Sendsmaily extends ApiClient
+{
+
+    public $baseUrl;
+    public $apiUser;
+    public $apiPassword;
+
+    public function subscribe(Model $model)
+    {
+
+        $result = $this->addSubscriber($model->email);
```

Comment:
as sendsmaily also accepts params related to email, please make it possible to send those